ui/appui-abstract/src/appui-abstract/utils/UiError.ts

```diff
@@ -9,7 +9,7 @@
 import { BentleyError, BentleyStatus, GetMetaDataFunction } from "@itwin/core-bentley";
 
 /** iTwin.js UI UiError class is a subclass of BentleyError. Errors are logged.
- * @public
+ * @public @deprecated in 4.3. Use [[Backend.BentleyError]] instead.
```

Comment:
Frontend packages dont typically expect to depend on `Backend` package, is this a typo (core?)